src/Components/Patient/PatientRegister.tsx

```diff
@@ -278,6 +280,8 @@ export const PatientRegister = (props: PatientRegisterProps) => {
     }
   };
 
+  const vaccineNames = ["CoviShield", "Covaxin"];
```

Comment:
add Select to the array, const vaccineNames = ["Select", "CoviShield", "Covaxin"];